src/components/layout/playground/applyForm.tsx

```diff
@@ -143,7 +148,17 @@ export const RequestDetails: React.FC<RequestProps> = ({ request }) => {
         </div>
         <SubtleBorder className="flex flex-col gap-1 p-8 min-w-fit bg-grey-background h-fit">
           <Field title="Category">{CATEGORY_LABELS[request.category]}</Field>
-          <Field title="Due date">{dueDateFormatted}</Field>
+          {hasNoDue ? (
+            <Field title={'Due Date'}>None</Field>
+          ) : (
+            <Field
+              title={`${
+                dueDateFormatted ? (isDue ? 'Was due' : 'Due') : 'Due'
+              }`}
+            >
+              {timeUntilDue ? dueDateFormatted : 'Today'}
+            </Field>
+          )}
```

Comment:
No need for nested if conditions here:
```suggestion
          <Field
            title={`${
              hasNoDue ? 'Due Date' : dueDateFormatted ? (isDue ? 'Was due' : 'Due') : 'Due'
            }`}
          >
            {hasNoDue ? 'None' : timeUntilDue ? dueDateFormatted : 'Today'}
          </Field>
```